package.json

```diff
@@ -3393,7 +3393,7 @@
       {
         "type": "clr",
         "when": "workspacePlatform == windows",
-        "hiddenWhen": "dotnet.debug.serviceBrokerAvailable",
+        "hiddenWhen": "true",
```

Comment:
NOTE: `true` works but I am not sure it is actually proper as it might just work by accident as I can also hide this just by putting something invalid here.

Reply:
`true` should be valid. They just check to see if the expression in the end just results in a true statement :D